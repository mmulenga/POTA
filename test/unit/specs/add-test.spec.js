/* Simple Jest test file to test out add() function in main.js */
/* Comment out this line in main.js first
  require('../node_modules/bootstrap-sass/assets/stylesheets/_bootstrap.scss'); */
/* Use command 'npm run unit' */
const add = require('@/main.js');

describe('add() in main.js', () => {
  it('should add two numbers', () => {
    const result = add(1, 2);
    const expected = 3;
    expect(result).toBe(expected);
  });
});

test('adds 1 + 2 to equal 3', () => {
  expect(add(1, 2)).toBe(3);
});
