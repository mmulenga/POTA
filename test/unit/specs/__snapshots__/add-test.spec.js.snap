// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`add() in main.js should match the snapshot 1`] = `3`;
